app/javascript/react/screens/App/Overview/components/Migrations/MigrationsInProgressCard.js

```diff
@@ -170,13 +170,18 @@ const MigrationsInProgressCard = ({
   // Tooltips
   const vmBarLabel = (
     <span>
-      <strong className="label-strong">{sprintf(__('%s of %s VMs'), completedVMs, totalVMs)}</strong> {__('migrated')}
+      <strong id="vms-migrated" className="label-strong">
```

Comment:
I think an `id` has to be unique in the page

Reply:
Negative. current one looks good. "vms-migrated" is good enough. no need to make it unique.

Reply:
🤔 oh hmmm yeah... also no spaces https://www.w3.org/TR/html5/dom.html#global-attributes  updating...

Reply:
so yeah, these id are unique to their tree, but unsure about the id with spaces... it doesn't seem to cause much trouble 🔨

Reply:
@kedark3 does an id with spaces cause you any grief? is this something we ought to change?

Reply:
@AllenBW Have not come across this issue before. So only time will tell. But I hope it won't cause any issues to have spaces.

Reply:
One last thought, and then I'll leave it alone

https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementById

> The id global attribute defines a unique identifier (ID) which must be unique in the whole document.

Also, if I try `document.getElementById('vms-migrated')`, it only ever returns the first one

Reply:
Yeah this makes sense, not unexpected, I believe when qe tests though they traverse down the tree, one reason why we need two ids here (I'll eat 🍝 if wrong @kedark3 😬 )